tex/paper.tex

```diff
@@ -329,27 +330,173 @@ \subsection{Description of the Axelrod python package}\label{sec:description-of-
+results from the full tournament.
+
+\begin{figure}[!hbtp]
+    \centering
+    \includegraphics[width=.8\textwidth]{../img/tournament.pdf}
+    \caption{Results from the library tournament (2015-12-19)}
+    \label{fig:tournament}
+\end{figure}
+
+The current winning strategy is a novel strategy of type: \textbf{LookerUp}. This
+is a strategy that maps a given set of states to actions. The state space is
+defined generically by \(m, n\) so as to map states to actions in the following
+way:
+
+\[(\underbrace{(C, D, D, D, C, D, D, C)}_{m\text{ first actions by opponent}},
+\overbrace{((C, C), (C, C))}^{n\text{ last pairs of actions}}) \to D\]
+
+The above example is an incomplete illustration of the mapping for \(m=8, n=2\).
+Intuitively this state space uses the initial plays of the opponent to gain some
+information about the intentions of it whilst still taking in to account the
+recent play.  The actual winning strategy an instance of the this framework for
+\(m=n=2\) for which an evolutionary algorithm has been used to train it. More
+details of this can be found in \cite{mojones}. In \cite{} experiments are
+carried out to evaluate how this particular strategy behaves in environments
+other than those in which it was trained\dots
+
+There are various other insights that have been gained from ongoing open
+research on the library. These include:
+
+\begin{itemize}
+    \item A closer look at zero determinant strategies, showing that
+        extortionate strategies obtain a large number of wins \textit{but} do
+        not perform particularly well. This is relevant given the findings of
+        \cite{Stewart2012} in which zero determinant strategies are shown to be
+        able to perform better than a given other strategy. This finding extends
+        to noisy tournaments (which are also implemented in the library).
+    \item This negative relationship between wins and performance does not
+        generalise, there are some strategies that perform well, both in terms
+        of matches won and overall performance: Back stabber, Double crosser and
+        Fool Me Once. These strategies continue to perform well in noisy
+        tournaments.
+    \item Strategies like LookerUp and MetaHunter seem to be generally
+        cooperative yet still exploit naive strategies. The MetaHunter strategy
+        is a particular type of Meta strategy which uses a variety of other
+        strategy behaviours to choose a best action. These strategies perform
+        very well in general and continue to do so in noisy tournaments.
+\end{itemize}
+
+Details for the above can be found at \cite{}.
+
+%Plot of memory_depth vs mean score, mean wins?
 
 \section{Conclusion}\label{sec:conclusion}
 
+This paper has presented a game theoretic software package that aims to address
+reproducibility of research in to the iterated prisoner's dilemma. Through the
+very nature of the development of this library, all open research and
```

Comment:
The open nature of the development of the library has lead rapidly to the inclusion of many well known strategies, many novel strategies, and new and recapitulated insights